lib/twine/formatters.rb

```diff
@@ -1,30 +1,13 @@
-require 'twine/formatters/abstract'
-require 'twine/formatters/android'
-require 'twine/formatters/apple'
-require 'twine/formatters/flash'
-require 'twine/formatters/gettext'
-require 'twine/formatters/jquery'
-require 'twine/formatters/django'
-require 'twine/formatters/tizen'
-
 module Twine
   module Formatters
-    @formatters = [Formatters::Apple, Formatters::Android, Formatters::Gettext, Formatters::JQuery, Formatters::Flash, Formatters::Django, Formatters::Tizen]
+    @formatters = []
 
     class << self
       attr_reader :formatters
-
-      ###
-      # registers a new formatter
-      #
-      # formatter_class - the class of the formatter to register
-      #
-      # returns array of active formatters
-      #
-      def register_formatter formatter_class
-        raise "#{formatter_class} already registered" if @formatters.include? formatter_class
-        @formatters << formatter_class
-      end
```

Comment:
I'm ok with breaking backward compatibility in the next major version of twine as long as there is a clear, documented upgrade path. It helps that I think that there are currently very few plugins in the wild.

However, if we can keep the old way around (and deprecate it) for the next version, that would be preferred.